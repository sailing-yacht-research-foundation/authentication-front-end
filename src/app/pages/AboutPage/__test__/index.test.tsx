import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { AboutPage } from '..';
// import i18next from 'i18next';

const shallowRenderer = createRenderer();

describe('<AboutPage />', () => {
  it('should render the home page and matchs the snapshot', () => {
    shallowRenderer.render(<AboutPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});