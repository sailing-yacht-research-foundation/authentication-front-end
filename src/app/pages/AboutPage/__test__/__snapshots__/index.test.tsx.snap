// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> should render the home page and matchs the snapshot 1`] = `
<AboutPage__Wrapper>
  <VideoSection />
  <BenefitCarousel />
  <Footer />
</AboutPage__Wrapper>
`;
